specs/paymentrequest.html

```diff
@@ -682,9 +689,21 @@ <h2 id="state-transitions" class="informative">State transitions</h2>
           <p>If the sequence only contains one item, then this is the shipping option that
           will be used and <a><code>shippingOption</code></a> will be set to the <code>id</code>
           of this option without running the <a>shipping option changed algorithm</a>.</p>
+          <p>If an item in the sequence has the <code>selected</code> field set to <code>true</code>,
+          then this is the shipping option that will be used by default and <a><code>shippingOption</code></a>
+          will be set to the <code>id</code> of this option without running the <a>shipping option changed
+          algorithm</a>. Authors SHOULD NOT set <code>selected</code> to <code>true</code> on more than
+          one item. If more than one item in the sequence has <code>selected</code> set to <code>true</code>,
+          then <a>user agents</a> MUST select the last one in the sequence.</p>
           <p>The <code>shippingOptions</code> field is only used if the <a><code>PaymentRequest</code></a> was
           constructed with <a><code>PaymentOptions</code></a> <code>requestShipping</code>
           set to <code>true</code>.</p>
+          <p class="note">
+            If the sequence contains only one item or if the sequence has an item with the <code>selected</code>
+            field set to <code>true</code>, then authors SHOULD ensure that the <code>total</code> field includes
```

Comment:
Doesn't this mean that developers will have to write different code paths for total calculation based on whether or not they have a default shipping option for the customer? Wouldn't it be easier if the browser just fired the "shippingoptionchange" event when the UI appears if the developer sets the selected field? The general question here is what would result in an easier implementation for the developer?